src/yogesh/Yogesh_Assignment54.java

```diff
@@ -0,0 +1,42 @@
+//Assignment-54 : 13th Oct'2023
+//
+//Write a program to return unique vowels included in the string, ensure sequence is maintained.
+//String str = "aakaniksha eaep dube";
+//output : iu
+//
+//String getVowels(String str){
+//	String output = "";
+//	while(str.length() != 0){
+//		
+//	}	
+//}
+package yogesh;
+
+public class Yogesh_Assignment54 {
+
+	String getVowvels(String str) {
+		String output = "";
+		int index = 0;
+		while (index < str.length()) {
+			char ch = str.charAt(index);
+			if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u') {
+				// if (str.indexOf(ch) >= 0) {
+				int firstIndex = str.indexOf(ch);
+				int lastIndex = str.lastIndexOf(ch);
+				if (firstIndex == lastIndex) {
+					output += ch;
+				}
+			}
```

Comment:
You can also write direclty in if condition as str.indexOf(ch)==str.lastIndexOf(ch)